en/includes/guides/account-configurations/user-onboarding/self-registration.md

```diff
@@ -173,3 +173,200 @@ Note the following details in the response payload:
 
 Once you have identified the `emailVerified` status of the user and the method by which the user is onboarded (self-registration or onboarded by an administrator), you can enforce any access restrictions for that user through your application logic.
```

Comment:
```suggestion
Once you have identified the `emailVerified` status of the user and the method used to onboard the user (either through self-registration or by an administrator), you can enforce any access restrictions through your application logic.
```

Reply:
This content is not something introduced with my improvment. However, will update this as well .

Reply:
Resolved via https://github.com/wso2/docs-is/pull/5355/commits/8aab8b464237b6ab3565afe5ecc80b52e7d3197a